src/board/generic_esp32/generic-esp32-board.c

```diff
@@ -81,7 +83,8 @@ void BoardInitPeriph(void)
         .mode = 0,
         .spics_io_num = lora_spi.cs,
         .command_bits = 8,
-        .address_bits = 16
+        .address_bits = 16,
+        .queue_size = 7
```

Comment:
why 7?

Reply:
you're right, can be 1. changing and added comment.